layouts/partials/article/_feedback.html

```diff
@@ -1,15 +1,18 @@
 <hr/>
 <div class="feedback block">
-  <h4 id="bug-reports-and-feedback">Bug Reports and Feedback</h4>
+  <h4 id="bug-reports-and-feedback">Support and feedback</h4>
   <p>
-    Thank you for being willing to help test InfluxDB v2.0 beta!
-    Feedback and bug reports are welcome and encouraged both for InfluxDB and this documentation.
-    Submit feedback using one of the following methods:
-  </p>
+    Thank you for being part of our community! 
+    We welcome and encourage your feedback and bug reports for InfluxDB and this documentation. 
+    To find support or submit feedback, the following resources are available:
+  </p>  
   <ul>
     <li><a href="https://community.influxdata.com/c/influxdb2" target="_blank">Post in the InfluxData Community</a></li>
     <li>In the <strong>InfluxDB UI</strong>, click <strong class="white-space:nowrap"><span class="icon-ui-nav-chat" style="font-size:1.25rem;margin:0 .25rem 0;"></span> Feedback</strong> in the left navigation bar.</li>
```

Comment:
The feedback functionality has been removed from the UI. We can take remove this line.